File: src/codes/Users_Manual/modulation/custom.rs
# use autd3::prelude::*;
# use autd3::modulation::Custom;
# #[allow(unused_variables)]
# fn main() -> Result<(), Box<dyn std::error::Error>> {
let m = Custom::new([0xFF, 0x00], 4000 * Hz)?;
# Ok(())
# }

File: src/codes/Users_Manual/modulation/custom.cpp
//~#include<vector>
//~#include<autd3.hpp>
#include <autd3/modulation/custom.hpp>

//~int main() {
autd3::modulation::Custom(std::vector<uint8_t>{0xFF, 0x00}, 4000u * autd3::Hz);
//~return 0; }